jobs/integr8ly/nightly-pipeline-qe-pony.yaml

```diff
@@ -0,0 +1,193 @@
+                            string(name: 'OC_PASSWORD', value: 'Password1')]
+                                
+                        // Waiting for 5 minutes so that resources scheduled for termination are actually terminated
+                        sleep time: 5, unit: 'MINUTES'
+                    } // stage
+                    
+                    stage ('Uninstall #2') {
+                        build job: 'uninstallation-pipeline-qe-pony', parameters: [
+                            string(name: 'REPOSITORY', value: "${INSTALLATION_REPOSITORY}"),
+                            string(name: 'BRANCH', value: "${INSTALLATION_BRANCH}"),
+                            string(name: 'CLUSTER_URL', value: "${CLUSTER_URL}"),
+                            string(name: 'ANSIBLE_USER', value: "${ANSIBLE_USER}"),
+                            string(name: 'MASTER_URLS', value: "${MASTER_URLS}"),
+                            string(name: 'OC_USER', value: "${OC_USER}"),
+                            string(name: 'OC_PASSWORD', value: "${OC_PASSWORD}")]
+
+                        // Waiting for 1 minute so that resources scheduled for termination are actually terminated
+                        // Since this is 2nd uninstall there is no need for a long wait here
+                        sleep time: 1, unit: 'MINUTES'
+                    } // stage
+
+                    stage('Install #1') {
+                        build job: 'installation-pipeline-qe-pony', parameters: [
+                            string(name: 'REPOSITORY', value: "${INSTALLATION_REPOSITORY}"),
+                            string(name: 'BRANCH', value: "${INSTALLATION_BRANCH}"),
+                            string(name: 'CLUSTER_URL', value: "${CLUSTER_URL}"),
+                            string(name: 'GH_CLIENT_ID', value: "${GH_CLIENT_ID}"),
+                            string(name: 'GH_CLIENT_SECRET', value: "${GH_CLIENT_SECRET}"),
+                            booleanParam(name: 'SELF_SIGNED_CERTS', value: Boolean.valueOf("${SELF_SIGNED_CERTS}")),
+                            string(name: 'ANSIBLE_USER', value: "${ANSIBLE_USER}"),
+                            string(name: 'MASTER_URLS', value: "${MASTER_URLS}"),
+                            string(name: 'OC_USER', value: "${OC_USER}"),
+                            string(name: 'OC_PASSWORD', value: "${OC_PASSWORD}")]
+                                
+                            sleep time: 3, unit: 'MINUTES'
+                    } // stage
+                    
+                    stage('Execute "After Installation" Workarounds #2') {
+                        if (INSTALLATION_BRANCH == 'master') {
+                            // this patches the webapp to use master image (v2.2.0 is used otherwise)
+                            sh """
+                                oc login https://master.${YOURCITY}.openshiftworkshop.com -u admin@example.com -p Password1  --insecure-skip-tls-verify
```

Comment:
I don't see **YOURCITY** in the parameter list

Reply:
Good catch. THanks